lib/network_utils.ml

```diff
@@ -759,6 +766,12 @@ module Dhclient = struct
     let conf = generate_conf ~ipv6 interface options in
     Xapi_stdext_unix.Unixext.write_string_to_file (conf_file ~ipv6 interface) conf
 
+  let remove_conf_file ?(ipv6=false) interface =
+    let file = conf_file ~ipv6 interface in
+    try
+      Unix.unlink file
+    with _ -> ()
```

Comment:
Maybe log this?
```
| exn-> warn "failed to remove file %s: %s" file (Printexc.to_string exn)
 ```

Reply:
If it fails, that's probably because the file doesn't exist, which is fine.